tests/wicked/startandstop_sut.pm

```diff
@@ -72,6 +72,17 @@ sub run {
     $self->cleanup($config, "br0");
     $self->cleanup($dummy,  "dummy0");
 
+    $self->before_scenario('Test 3', 'Bridge - ifup, remove all config, ifreload', $iface);
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifup');
+    assert_script_run("rm /etc/sysconfig/network/ifcfg-$iface $config $dummy");
+    assert_script_run("wicked ifreload --timeout infinite all");
+    my $res = script_output("ip link|grep 'dummy0\|br0'||true");
```

Comment:
this PR already hit osd https://github.com/os-autoinst/os-autoinst/pull/1030 so you may also use this 
`my $res = script_output("ip link|grep 'dummy0\|br0'",90, proceed_on_failure => 1);`

Reply:
you could also use just script_run()

Reply:
yeah actually in this case probably better just to use `script_run`

Reply:
done